acceptance-tests/upgrade/update_and_upgrade_aurora_postgresql_test.go

```diff
@@ -72,6 +72,13 @@ var _ = Describe("UpgradeAuroraPostgreSQLTest", Label("aurora-postgresql", "upgr
 			got = appTwo.GET("%s/%s", schema, keyOne).String()
 			Expect(got).To(Equal(valueOne))
 
+			By("deleting bindings created before the upgrade")
+			bindingOne.Unbind()
+
+			By("creating new bindings and testing they still work")
+			serviceInstance.Bind(appOne)
+			apps.Restage(appOne)
+
 			By("updating the instance version")
 			serviceInstance.Update(services.WithParameters(map[string]any{
```

Comment:
I notice that we keep the update parameter here. Is that deliberate?

Reply:
I think i might have missed it